### PR TITLE
Update mtr-device-config-account-include.md

### PR DESCRIPTION
**Pull request by rededwin**
Add GO link to Teams Rooms advanced deployment guide within Admin Center

### CODE REVIEW

File: Teams/includes/mtr-device-config-account-include.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ When you create the resource mailbox, you can specify whether you want to allow
 > [!TIP]
 > We recommend that you create all resource accounts using Exchange Online and Azure Active Directory.
 
+> [!TIP]
+> You can automatically configure recommended Teams Rooms resource settings via the [Surface Hub and Microsoft Teams Rooms automated setup guide](https://go.microsoft.com/fwlink/?linkid=2221605).
```
Comment (JohanFreelancer9) [inline Teams/includes/mtr-device-config-account-include.md:15]:
```suggestion
>
> You can automatically configure recommended Teams Rooms resource settings via the [Surface Hub and Microsoft Teams Rooms automated setup guide](https://go.microsoft.com/fwlink/?linkid=2221605).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rededwin, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by rededwin** [comment]
@dstrome when will this be published to public?

**Comment by dstrome** [comment]
Sorry, updates to includes don't trigger a refresh of the articles they're in. I've started a full build of the repo which will refresh all the content on the site, including the articles this include is in. Should be live in 20-30 mins.

**Pull request merged**